core-ui/src/components/BusolaExtensions/BusolaExtensionList.js

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import { useTranslation, Trans } from 'react-i18next';
+import LuigiClient from '@luigi-project/client';
+
+import { ResourcesList } from 'shared/components/ResourcesList/ResourcesList';
+import { Link } from 'shared/components/Link/Link';
+
+import { BusolaExtensionCreate } from './BusolaExtensionCreate';
+
+export function BusolaPluginList(props) {
+  const { t } = useTranslation();
+
+  const customColumns = [
+    {
+      header: t('common.headers.namespace'),
+      value: resource => resource.metadata.namespace,
+    },
+  ];
+
+  const description = (
+    <Trans i18nKey="extensibility.extensions.description">
+      <Link
+        className="fd-link"
+        url="https://github.com/kyma-project/busola/tree/main/docs/extensibility"
+      />
+    </Trans>
+  );
+
+  return (
+    <ResourcesList
```

Comment:
Do we want to keep the `config map` name or should we rename it to `extensions` (to match the label in the left-side navigation and not confuse the user)?

Reply:
it was supposed to be "extensions". turns out we've had an inconsistency in API between `ResourceList` and `ResourceDetails`.